light-zk.js/src/wallet/user.ts

```diff
@@ -542,7 +542,12 @@ export class User {
             [this.provider.wallet!.publicKey],
           ),
         );
-
+        console.log("transaction: ", transaction);
+        transaction.recentBlockhash = (
```

Comment:
this shouldn't be necessary, pls confirm and remove

Reply:
same as the one above => done!